etc/proxy-server.conf-sample

```diff
@@ -131,6 +131,9 @@ use = egg:swift3#swift3
 # upload, except the last part.
 min_segment_size = 5242880
 
+# Expose S3API method in swift log
+#log_s3api = False
```

Comment:
This is ambiguous. Does this refer to "the S3 API", to the "s3api" command of AWS Command Line Interface, or to OpenStack Swift's "s3api" middleware?

If this refers to the "s3api" command of AWS Command Line Interface, I would rename the parameter to `log_s3api_command`, since the documentation talks about "commands".

If this refers to "the S3 API", I would rename the parameter to `log_s3_api_operation` (or `log_s3_operation`) since the documentation talks about "operations". But in that case, to be coherent, all logged operations must be changed to their camelcase equivalent.